src/Neo.Extensions/RandomExtensions.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// RandomExtensions.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using System;
+using System.Numerics;
+
+namespace Neo.Extensions
+{
+    public static class RandomExtensions
+    {
+        public static BigInteger NextBigInteger(this Random rand, int sizeInBits)
+        {
+            if (sizeInBits < 0)
```

Comment:
quick question, if it must be positive, why not ushort

Reply:
I'm just moving the function to extensions library.